.env.example

```diff
@@ -4,4 +4,13 @@ nginx_server_name=localhost
 nginx_ssl=0
 POSTGRES_PASSWORD=appyter
 MINIO_ACCESS_KEY=access
-MINIO_SECRET_KEY=secret
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Looks like you didn't resolve some conflicts

Reply:
I think this should all be updated now...let me know if it looks right